### ISSUE TITLE
wsdl2go panicking

### ISSUE THREAD

**Issue by lalanne** [fiorix/wsdl2go#67]
Hello, 
executing wsdl2go with a valid wsdl shows me the following error:
```
./wsdl2go < perfilamiento.wsdl > perf1.go                                                                                                                                                                                                                               
panic: interface conversion: http.RoundTripper is nil, not *http.Transport

goroutine 1 [running]:
panic(0x7ff9a0, 0xc820118480)
        /usr/lib/go-1.6/src/runtime/panic.go:481 +0x3e6
main.main()
        /home/lalanne/go/src/github.com/fiorix/wsdl2go/main.go:53 +0x5bd
```

any ideas?
Thanks

**Comment by turboezh** [comment]
Duplicate of #64

**Comment by fiorix** [comment]
Fixed in e1518c1
